cmd/tapcli/universe.go

```diff
@@ -70,6 +71,11 @@ var universeRootsCommand = cli.Command{
 				"either 'issuance' or 'transfer'",
 			Value: universe.ProofTypeIssuance.String(),
 		},
+		cli.BoolFlag{
+			Name: skipAmountsByIdName,
+			Usage: "skip showing the amounts by ID for grouped " +
+				"assets to optimize response size and speed",
+		},
 	},
```

Comment:
Doesn't this default to `false`? And so we wont skip? I think our intent is to skip by default. I think the flag should capture the positive case here: `withAmountsById`.

Reply:
> But I think we do want the CLI to remain at the previous behavior, so I'm going to fix that.

I see this comment from you now.